cross/fdupes/Makefile

```diff
@@ -0,0 +1,34 @@
+PKG_NAME = fdupes
+
+NAME = fdupes
+PKG_DIST_SITE = https://github.com/adrianlopezroche/fdupes.git
+# From Makefile.inc/VERSION in the git checkout
+PKG_VERSION = 1.51
+PKG_GIT_HASH = f7312fa
+PKG_DOWNLOAD_METHOD = git
+
+PKG_EXT = tar.gz
+PKG_DIST_FILE = $(PKG_NAME)-git$(PKG_GIT_HASH).$(PKG_EXT)
+PKG_DIR = $(PKG_NAME)-git$(PKG_GIT_HASH)
+
+DEPENDS =
+
+HOMEPAGE = https://github.com/adrianlopezroche/fdupes
+COMMENT  = FDUPES is a program for identifying or deleting duplicate files residing within specified directories.
+LICENSE  = MIT
+
+CONFIGURE_TARGET = nop
+COMPILE_TARGET = myCompile
+INSTALL_TARGET = myInstall
+
+include ../../mk/spksrc.cross-cc.mk
+
+.PHONY: myCompile
+myCompile:
+	$(RUN) $(TC_PATH)$(TC_PREFIX)gcc $(CFLAGS) -D'VERSION=$(PKG_VERSION)' -o $(SRC_DIR)fdupes $(SRC_DIR)fdupes.c $(SRC_DIR)md5/md5.c $(LIBS)
```

Comment:
Why not use the method as described in the INSTALL document: https://github.com/adrianlopezroche/fdupes/blob/master/INSTALL?